### ISSUE TITLE
Sonoma

### ISSUE THREAD

**Issue by dclive** [Wamphyre/HP-ELITEDESK-800-G2-DM-MINI-HACKINTOSH-35W#3]
Is there anything we should consider about Sonoma upgrades?  Flip to iMacPro SMBIOS then upgrade, after going to .97 and the latest KEXTs?

**Comment by dclive** [comment]
For everyone's awareness, in OCAT simply selecting the iMacPro SMBIOS, and then telling it to generate all numbers, allows me to then install Sonoma with no issues. 

More detail:  Installed Ventura first to check and test setup.  Had to disable VT-d in BIOS.  After 13.5 was installed, installed 14.3 via download from Apple (namely, download the 13GB file, then install it by double-click; install via General / Updates didn't work for me).  

I can't get wifi to connect to my WAPs.  No wifi functionality whatsoever.  I have the default/ stock Intel wifi modules.  Flipping to itwlm 2.3alpha releases doesn't change anything...no go.  The wifi card sees my local WANs but doesn't allow me to actually connect.  I type in the password, it thinks for a bit, then .. never connects.

**Comment by Camouflagge** [comment]
I also took the itlwm kext and WiFi is not possible, I use the Helpport APP.


 Does hibernation work for you?

**Comment by dclive** [comment]
No.  But that’s expected; I understand it’s never worked.  I’m not sure if it’s “never for this model” or “never for HD530 graphics” but that part was no surprise.  

Fan noise on mine is astronomical.  Given that it’s 8 years old or so I’m not surprised but wow, it’s loud.  

But it runs really well overall - 2 DisplayPort outs both work.

**Comment by Camouflagge** [comment]
How did you solve the WiFi problem?

**Comment by dclive** [comment]
Using USB-based ethernet.  My internet NIC is not reliable, but that, too, is a known issue with that particular machine (ie a local hardware issue).

**Comment by lalaRLH** [comment]
I have a testing version of kext for the WiFi will post once I’m home and in front of the machine.

**Comment by lalaRLH** [comment]
Uploading my EFI folder on Sonoma latest OS version. I took the WiFi alpha build and then did a bit of tweaking here and there. Note you will need to go into a tool such as OCAT and populate System Serial Number, MLB, System UUID and ROM here. 
![Screenshot 2024-02-06 at 4 06 53 am](https://github.com/Wamphyre/HP-ELITEDESK-800-G2-DM-MINI-HACKINTOSH-35W/assets/154371240/9f345aa8-9d9f-4636-a29d-3bb71f38ec0a)

Also generate APECID in misc - security: you can do this in a terminal window by typing 
python3 -c 'import secrets; print(secrets.randbits(64))'

Notes: I rebuilt under latest Opencore versions. I redeployed all kext files to latest. For the ACPI I noted the ones provided were not in standardised ACPI format with a bunch of unexpected NULL values and other random characters which the human eye cannot interpret (nor can reputable AVs). I didn't like that, so they got replaced with versions that are aligned to the standard and have very clear visible text throughout that leaves no mystery. 

Added tweaks for hibernation but they are still very hit and miss. Updated to newer iMac model that aligned with compatibility checks and allowed newer OS.

Note: For users who are not connected via a same side connector e.g. Mini DP to HDMI instead of MiniDP to MiniDP - this cannot boot into the OS setup. If used more broadly once OS is installed the displays often go purple due to mismatching of the interfaces. 

Due to 25MB limit I've sent via secure storage facility:
https://send.vis.ee/download/8957f264d64234ef/#1yiIU5lPLhFvzU_3wAKL0Q

**Comment by dclive** [comment]
@lalaRLH 

> Due to 25MB limit I've sent via secure storage facility: https://send.vis.ee/download/8957f264d64234ef/#1yiIU5lPLhFvzU_3wAKL0Q

Can you repost this?  It appears the file has timed out / disappeared.
